### PR TITLE
EZP-31617 - Enabling Personalization

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | EZP-31617 |
| Versions      | master |

Add a topic that describes how you must proceed to enable the Personalization feature.

### CODE REVIEW

File: docs/guide/personalization/enabling_personalization.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Enabling Personalization
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:1]: Q: should we capitalize Personalization throughout?

File: docs/guide/personalization/enabling_personalization.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+The Ibexa Personalization solution is based on a client-server architecture.
+To make it operational, you must obtain and set up certain authentication parameters.
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:4]:
```suggestion
To enable it, you must get and set up authentication parameters.
```

File: docs/guide/personalization/enabling_personalization.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Requesting access to the server
+
+At first you must request credentials required to access the recommendation server.
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:8]:
```suggestion
First you must request credentials to access the recommendation server.
```

File: docs/guide/personalization/enabling_personalization.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+To do this, in the Back Office, select **Personalization** and then **Dashboard**.
+On the welcome screen, enter the following details in their respective fields:
+
+* a full name of the person responsible for accepting the terms and conditions of the Personalization service
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:13]:
```suggestion
- full name of the person responsible for accepting the terms and conditions of the Personalization service
```

File: docs/guide/personalization/enabling_personalization.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+* an e-mail address to which you want the credentials to be sent to
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:14]:
```suggestion
- email address to which you want the credentials to be sent
```

File: docs/guide/personalization/enabling_personalization.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+* an Installation key that can be found on the **Maintenance and Support agreement details** page in the service portal
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:15]:
```suggestion
- installation key that can be found on the **Maintenance and Support agreement details** page in the service portal
```

File: docs/guide/personalization/enabling_personalization.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+Select a checkbox to confirm that you've read the terms and conditions, and then click **Submit**.
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:17]:
```suggestion
Accept the terms and conditions, and then click **Submit**.
```

File: docs/guide/personalization/enabling_personalization.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+Your request is sent to Ibexa, and you will receive a confirmation e-mail in response.
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:18]:
```suggestion
Your request is sent to Ibexa, and you will receive a confirmation email in response.
```

File: docs/guide/personalization/enabling_personalization.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+## Configuring mandator credentials
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:20]: Idea for future - is the term "mandator" used in the UI? It's counter-intuitive, so maybe we could replace it with a different term in customer-facing texts?

File: docs/guide/personalization/enabling_personalization.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+When you've received the confirmation e-mail, you must add the credentials to your configuration.
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:22]:
```suggestion
When you receive the confirmation email, you must add the credentials to your configuration.
```

File: docs/guide/personalization/enabling_personalization.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+In the root folder of your project, edit either the `.env` or `.env.local` file by adding two lines that 
+may look similar to the following example. 
+
+```
+RECOMMENDATION_CUSTOMER_ID=12345
+RECOMMENDATION_LICENSE_KEY=67890-1234-5678-90123-4567
+```
+
+You can now go back to the Back Office, refresh the personalization dashboard and proceed to configuring
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:31]:
```suggestion
You can now go back to the Back Office, refresh the Personalization dashboard and proceed to configuring 
```
?

File: docs/guide/personalization/enabling_personalization.md [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+```
+RECOMMENDATION_CUSTOMER_ID=12345
+RECOMMENDATION_LICENSE_KEY=67890-1234-5678-90123-4567
+```
+
+You can now go back to the Back Office, refresh the personalization dashboard and proceed to configuring 
+the Personalization solution.
+
+## Changing the Installation key
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:34]:
```suggestion
## Changing the installation key
```

File: docs/guide/personalization/enabling_personalization.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+
+If necessary, you can modify the Installation key configured in the personalization settings.
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:36]:
```suggestion
If necessary, you can modify the installation key configured in the personalization settings.
```

File: docs/guide/personalization/enabling_personalization.md [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+
+  Clearing the **Installation key** field disables the Personalization solution for your account temporarily.
+  Entering the same key re-enables the feature.
+
+## Unregistering the installation key
+
+You can also revoke your agreement to the terms and conditions of the Personalization solution and 
+permanently disable the feature for your account.
+
+To do this, use your browser to navigate to an address similar to the following example:
```
Comment (DominikaK) [inline docs/guide/personalization/enabling_personalization.md:50]:
```suggestion
To do this, navigate to the following address:
```

**Review by DominikaK — CHANGES_REQUESTED** [review]

File: docs/guide/personalization/enabling_personalization.md (hunk 9 above)
Comment (dabrt) [reply docs/guide/personalization/enabling_personalization.md:20]: I'd personally go with "user". However, I've checked that there are occurrences in the doc, and Damian seems to like using it.

File: docs/guide/personalization/enabling_personalization.md (hunk 6 above)
Comment (dabrt) [reply docs/guide/personalization/enabling_personalization.md:15]: A capitalized version of "Installation key" happened elsewhere in the docs, hence here.
https://www.google.com/search?q=site:https://doc.ibexa.co+%22installation+key%22

Comment (DominikaK) [reply docs/guide/personalization/enabling_personalization.md]: Hm, ideally both should be lowercase, but let's keep it as is for now for consistency.

**Review by DominikaK — APPROVED** [review]

**Review by Nattfarinn — APPROVED** [review]

File: docs/guide/personalization/enabling_personalization.md (hunk 9 above)
Comment (dkorostynski) [reply docs/guide/personalization/enabling_personalization.md:20]: it is not about liking but rather consistency

**Review by dkorostynski — APPROVED** [review]

**Pull request merged**